Resources/Prototypes/Catalog/LatheRecipes/misc.yml

```diff
@@ -57,12 +57,11 @@
   completetime: 500
   materials:
     steel: 80
-    glass: 30
```

Comment:
Why'd you take glass out here? Lanterns have glass.

Reply:
that's the fire extinguisher.